pkg/kubelet/kubelet.go

```diff
@@ -463,10 +464,19 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	}
 	glog.Infof("Hairpin mode set to %q", klet.hairpinMode)
 
-	if plug, err := network.InitNetworkPlugin(kubeDeps.NetworkPlugins, kubeCfg.NetworkPluginName, &networkHost{klet}, klet.hairpinMode, klet.nonMasqueradeCIDR, int(kubeCfg.NetworkPluginMTU)); err != nil {
-		return nil, err
+	if kubeCfg.ExperimentalRuntimeIntegrationType == "" {
```

Comment:
Because of two reasons:

1. Network plugin has been moved to runtime inside this PR.
2. Only dockershim depends on the `ExperimentalRuntimeIntegrationType` flag, other CRI runtimes will be integrated with explicit runtime/image service endpint.